### ISSUE TITLE
The pubspec is limiting dart to 2.0 which is making my projects 2.1.0 fail

### ISSUE THREAD

**Issue by chetstriker** [ianldgs/material_search#30]
The pubspec line in question, does this need to be restrictive?
environment:
sdk: ">=1.19.0 <2.0.0"

**Comment by modulovalue** [comment]
i'm also having problems, please update the constraints 
thanks

**Comment by kezzbr** [comment]
Following this suggestion too. Hope this guys pay attention to it.

**Comment by modulovalue** [comment]
```
Upper constraints on the SDK version
Once your package passes Dart 2 analysis, update the upper constraint to declare that the package is compatible with Dart 2:

environment:
  # Works in Dart 2 only.
  sdk: '>=2.0.0 <3.0.0'
```
[source](https://www.dartlang.org/dart-2/)

these constraints should probably be better if you don't want to update them often

**Comment by ianldgs** [comment]
Just released 0.2.8 on pub with the fix from @naft-a. Thanks everyone!
